04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -239,9 +264,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Saving results to remote database\n",
+    "<a class=\"anchor\" id=\"savemydb\"></a>\n",
+    "# Saving results to remote database MyDB\n",
     "\n",
-    "Alternatively we may want to store the results in a table called <i>mags</i> in our remote database."
+    "MyDB is a useful storage space for users to save their work as a table. It can only store data tables.\n",
+    "\n",
+    "Here we show how to store the results in a table called *mags* in our remote database MyDB."
```

Comment:
These two sentences should be probably combined. I.e. i) state that MyDB os remote per-user relational database, ii) that it can hold tables, iii) and that results of queries can be saved directly to MyDB. Then show example.